packages/material-ui-benchmark/package.json

```diff
@@ -28,6 +28,7 @@
     "@emotion/core": "^10.0.0",
     "@emotion/styled": "^10.0.0",
     "benchmark": "^2.1.4",
+    "emotion": "^10.0.7",
```

Comment:
@oliviertassinari I think this is now a more realistic benchmark. `emotion-server` should use `emotion@10` but it was using `emotion@9` before this change. `emotion@9` is a direct dependency of `react-select` and was therefore available for `emotion-server`.

Reply:
We should be using emotion@10 exclusively. I can fix that later on.